cloud/passportauth/src/auth/PassportAuth.ts

```diff
@@ -107,6 +108,21 @@ export class PassportAuth implements EndpointSecurity {
   }
 
   /**
+   * A middleware which checks if the user is authenticated.
+   * If the user is already authenticated, the user is redirected back to the application,
+   * otherwise it will proceed onto authenticating the user.
+   */
+  public isAuthenticated() {
```

Comment:
How about wrapping `isAuthenticated` and authService.authenticate into single middleware?